src/librustdoc/clean/mod.rs

```diff
@@ -628,6 +747,15 @@ impl Attributes {
                                 Err(e) => diagnostic.span_err(e.span, e.msg),
                             }
                             return None;
+                        } else if let Some(filename) = Attributes::extract_include(&mi) {
+                            match Attributes::load_include(include_path, &filename) {
+                                Ok(frag) => {
+                                    let line = doc_line;
+                                    doc_line += frag.len();
```

Comment:
Oh so you decided to put in the "real" line number instead of the one from the original file?

Reply:
Yeah. My thought was that if you're trying to trace a line number from the total markdown blob, then you could use that to get the doc fragment, since each attribute is joined with a line break anyway.